src/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@300&display=swap" rel="stylesheet">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <title>Laughs</title>
+</head>
+
+<body>
+  <ul class="topnav">
+    <li>
+      <!--htmlhint disable-next-line -->
+      <img class="logo" alt="Logo" src="#">
+    </li>
+    <li><a class="active" href="Home.html">Laughs</a></li>
+    <li><a href="Page1.html">Search Jokes</a></li>
+    <li><a href="Page2.html">About Laughs</a></li>
+  </ul>
+  <section class="jokes">
+    <div class="jokesFilters">
+      <h3>Jokes: <span id="numberJokes"></span></h3>
+    </div>
+    <div class="jokesGrid"></div>
+  </section>
```

Comment:
**Changes Required**
- [ ] The count of the jokes should be part of the navigation as seen in the given mock up desing i.e `Laughs(10)`. This will ensure that we are following the given design as close as possible. Let us try to make these changes.

Reply:
Done ✔

Reply:
Addressed this